package.json

```diff
@@ -78,6 +78,7 @@
     "eslint-plugin-notice": "0.9.10",
     "eslint-plugin-react": "7.26.1",
     "eslint-plugin-react-hooks": "4.2.0",
+    "fuzzyjs": "5.0.1",
```

Comment:
What are the reasons for adding `fuzzyjs` and is regular filtering insufficient? Is it recommended that consumers also use this dependency when using `Combobox`?

Reply:
@Francois-Esquire can we revert to what you had before? The `fuzzyjs` is not available to codesandbox, and we want to keep the raw demo library set to the bare minimum.

Reply:
@jzempel no problem, I've updated and reverted to how it was previously.

@hzhu to give some context, @jzempel and I were discussing fuzzy search and after some research into this, I figured I'd rely on a pre-existing solution with `fuzzyjs`.